frontend/src/views/Home.vue

```diff
@@ -151,16 +52,16 @@ export default {
         },
         {
           name: "Sponsors",
-          number: this.sponsors.length
+          number: "10"
         }
-      ];
-      return statsArray;
-    }
-  },
-  methods: {
-    ...mapActions({
-      login: "login/login"
-    })
+      ]
+  }),
+  computed: {
+    // GET COMMUNITY AND SPONSOR LIST FROM THE STORE
```

Comment:
why do you write in ALL-CAPS? BECAUSE THIS SHOULD BE YELLED? 🤣

Reply:
SO THAT IT'S EASIER TO SEE